blocks/header/header.js

```diff
@@ -142,42 +155,41 @@ const getSearchCategoryDropDown = (fragment) => {
 };
 
 /**
- * Build the search results popup
+ * Build the search results popup to reset the result set
  */
-const buildSearchResultsPopup = (block) => {
-  const searchResultsPopup = document.createElement('div');
-  searchResultsPopup.className = 'search-results-popup';
-  searchResultsPopup.id = 'search-results-popup';
-  const searchResultsPopupContainer = document.createElement('div');
-  searchResultsPopupContainer.className = 'search-results-popup-container';
+const buildSearchResultsPopup = (searchResultsPopupContainer) => {
   searchResultsPopupContainer.innerHTML = `
-    <div class="category equity">
+    <div class="loader hidden">
+      Searching...
+    </div>
+    <div class="category equity hidden">
       <span>Equity</span>
       <ul class="equity-list"></ul>
     </div>
-    <div class="category mf">
+    <div class="category mf hidden">
```

Comment:
Nit: classname can be mutual-fund as complete name was used in all categories

Reply:
have kept it intentionally to match the category keyword being passed in the API as much as possible